master/internal/sproto/task.go

```diff
@@ -15,13 +15,15 @@ type (
 	// AllocateRequest notifies resource managers to assign resources to a task.
 	AllocateRequest struct {
 		// Identifying information.
-		AllocationID model.AllocationID
-		TaskID       model.TaskID
-		JobID        *model.JobID
-		State        job.SchedulingState
-		Name         string
-		TaskActor    *actor.Ref
-		Group        *actor.Ref
+		AllocationID      model.AllocationID
+		TaskID            model.TaskID
+		JobID             *model.JobID
+		JobSubmissionTime *time.Time // TODO or a job info struct
```

Comment:
i think we should either cleanup these todos or make a ticket for them

Reply:
since I didn't have clear stories for addressing them I just cleaned them up